doc/en/contents.rst

```diff
@@ -10,32 +10,51 @@ Full pytest documentation
 .. toctree::
    :maxdepth: 2
 
-   overview
-   apiref
-   example/index
+   getting-started
```

Comment:
Not sure, I thought they meant a complete overhaul of the docs, including rewriting sections. Here I was interested in getting zero warnings, and to do that I had to include all documents in the table of contents, but which is something we already had agreed would be for the best (flat better than nested, etc)